flashinfer/gemm/gemm_base.py

```
@@ -2400,6 +2401,361 @@ def mm_fp8(
+    else:
+        raise ValueError(f"Unsupported SM major version: {sm_major}")
+
+
+def _check_mm_mxfp8_problem_size(
+    a: torch.Tensor,
+    b: torch.Tensor,
+    a_descale: torch.Tensor,
+    b_descale: torch.Tensor,
+    out: Optional[torch.Tensor] = None,
+    out_dtype: torch.dtype = torch.bfloat16,
+    backend: Literal["cutlass", "auto"] = "auto",  # unused
```

Ruff (0.14.14)

[warning] Unused function argument: `backend`

(ARG001)